extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -159,6 +160,15 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
     return this.autoFormat_ == 'rspv';
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  hasValidAdSenseHeight_() {
+    return Number(this.element.getAttribute('height')) ==
```

Comment:
Can you user.error when hasValidAdSenseHeight_ is false to ensure there is an actionable error in the console?  Also, there is no need to convert to Number as == is a type-less comparison (as opposed to === which is typed).  So, '320' == 320 will return true.